guides/assets/stylesrc/_main.scss

```diff
@@ -0,0 +1,717 @@
+body.guide {
+
+  -webkit-tap-highlight-color:rgba(38, 27, 35, 0);
+
+  // ----------------------------------------------------------------------------
+  // Typographic Baseline
+  // ----------------------------------------------------------------------------
+  
+  :root {
+    font-family: Inter, sans-serif;
+    font-feature-settings: 'liga' 1, 'calt' 1; /* fix for Chrome */
+  }
+  @supports (font-variation-settings: normal) {
+    :root { font-family: InterVariable, sans-serif; }
+  }
+
+  font-family: Inter, sans-serif;
+  font-size: 1.125rem; /* 18px */
+  font-style: normal;
+  font-weight: 400;
+  font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  -webkit-font-smoothing: antialiased;
+  line-height: 1.4;
+
+  font-feature-settings: "liga" 1;
+  @supports (font-variant-ligatures: common-ligatures) {
+    font-feature-settings: normal;
+    font-variant-ligatures: common-ligatures;
+  }
+
+  font-feature-settings: "onum" 1;
+  @supports (font-variant-numeric: oldstyle-nums) {
+    font-feature-settings: normal;
+    font-variant-numeric: oldstyle-nums;
+  }
+
+  p {
+    // Set hyphenation rules
+    hyphenate-limit-lines: 2; /* Safari */
+    -webkit-hyphenate-limit-before: 3;
+    -webkit-hyphenate-limit-after: 2;
+    hyphenate-limit-chars: 6 3 2; /* Not Safari */
+    hyphenate-limit-last: always;
+
+    @include media('>tablet') {
+      // On larger screens, set a width of 45-75 characters, using em as ch 
+      // has too much variation between various fonts
+      min-width: 23em;
+      max-width: 38em;
+    }
+
+    margin: 0 0 0.75em 0; // Space after paragraph
+    text-align: left;
+
+  } // p
+
+  // Links
+
+  a {
+
+    &:link {
+      color: $rf-brand;
+      text-decoration: underline;
+    }
+
+    &:hover,
+    &:active {
+      color: $rf-brand-dark;
+    }
+
+    &:visited {
+      color: $gray-600;
```

Comment:
The current guides do not keep a different visited style as far as I can tell, I'm guessing this is intentional?

![Screenshot 2024-03-18 at 16 56 51](https://github.com/rails/rails/assets/26328/85a2a01d-c54d-4879-ba59-a2d2f39511ff)

(not suggesting I have a preference, just pointing out a difference I noticed)

Reply:
Best practice would be to have a visited color so people know where they have been. Not a hill I'm planning to die on :) Would you all like to change it to be the same as `a:link`?

Reply:
No strong opinions here, I'll let others comment as well. Maybe ask Amanda too.

Reply:
I've also gone back and forth on this a lot. After clicking around I noticed the gray and was mildly annoyed by it for aesthetic reasons, but of course I also understand the adage of 'this is how I know where I have been'. I'd be curious to ask a broader audience. Will tag you @jathayde.

Reply:
Perhaps a much more subtle distinction would be a better fit?

As reference material that's expected to be repeatedly visited over the course of years, it seems unfortunate to over-emphasize visitedness, even if it is helpful for a theoretical new visitor who's trying to remember which single page they were reading the other day / is working their way through every guide in turn.

Reply:
Good point @matthewd. Maybe it's the jump from red to gray that sits strange with me. A subtle change would be great - those who need it can still see their path, but it won't stand out that much.

Reply:
Yeah, a more subtle distinction could work well :+1:

Reply:
Went with light for hover (5% lighter) and darker for visited (20% darker) in 3740edb9 and 3632e453